package.json

```diff
@@ -39,7 +39,7 @@
     "run-sequence": "^1.1.0"
   },
   "dependencies": {
-    "bluebird": "^2.9.27",
+    "bluebird": "2.9.30",
```

Comment:
I added a note to myself to remember unlocking this dependency once the issue is fixed: https://github.com/petkaantonov/bluebird/issues/689.